data/src/test/kotlin/cmm/apps/esmorga/data/event/EventRepositoryImplTest.kt

```diff
@@ -115,4 +115,22 @@ class EventRepositoryImplTest {
         Assert.assertEquals(localName, result[0].name)
     }
 
+    @Test
+    fun `given events locally cached when join event is requested then local events are cached`() = runTest {
+        val localEvent = listOf(EventDataMock.provideEventDataModel("localName"))
+        val eventId = localEvent.first().dataId
+
+        coEvery { userDS.getUser() } returns UserDataMock.provideUserDataModel()
```

Comment:
you do not need to add all these mocks, right? They are not used in the function you are testing.